.gitignore

```diff
@@ -42,3 +42,7 @@ coverage/
 /.gemtags
 
 public/sitemap.xml.gz
+Gemfile.lock
```

Comment:
Not a good idea to remove the Gemfile.lock file. Probably me messing up the rebase from develop

Also .env.example needs to be checked in so that new developers know what to copy across to .env